tests/test_customer_can_tie_payment_type_to_order.py

```diff
@@ -3,53 +3,75 @@
-	    self.paymentmanager = PaymentManager()
-	    self.order = Order(self.bobby)
-	    self.orderfinalizer = OrderFinalizer()
-	    self.ordermanager = OrderManager()
-
-	def test_check_that_an_order_is_empty(self):
-		"""
-		purpose: test to check if the cart is empty
-		author: Ike
-		methods: get_ordered_products: returns a list of products
-		"""
-		product = ProductOnOrderManager()
-		product.remove_all_products_from_order()
-		self.assertEqual(product.get_products_on_order(), [])
-
-	def test_check_that_order_can_be_completed(self):
-		"""
-		purpose: test to check if the cart is empty
-		author: Ike
-		methods: get_ordered_products: returns a list of products
-		"""
-		product = OrderManager()
-		product.products=['apples', 'oranges', 'ben and jerrys']
-		self.order.payment_type.append(self.paymentmanager.add_payment_type(self.bobby, "VISA", "123467"))
-
-		self.assertNotEqual(product.get_products_on_order(), [])
-
-		self.assertIn(self.paymentmanager.add_payment_type(self.bobby, "VISA", "123467"), self.order.payment_type)
+    """
+        purpose:
+        tests for an finalization of an order
+        via tying a payment type to an order
+        author: Ike
+        methods: test_products_have_been_added
+    """
+
+    def test_check_that_an_order_is_empty(self):
+        """
+        purpose: test to check if the cart is empty
+        author: Ike
+        methods: get_ordered_products: returns a list of products
+        """
+        jfk = Customer("John Kennedy", '1819 Heron Pointe Dr', "Nashville", "TN", 37214, '8889878888')
+        manage_status = CustomerStatusManager()
+        make_jfk_active_user = manage_status.change_status(jfk)
+        start_jfk_order = Order(jfk)
+        manage_order = OrderManager()
+        jfk_order = manage_order.create_order(start_jfk_order) #an order is created in the database
+        manage_products_on_order = ProductOnOrderManager()
+        current_order = manage_products_on_order.get_all_products_on_order() #get all products on active user's order
+        finalize_order = OrderFinalizer()
+        active_order_status = finalize_order.check_cart_contains_items()
+        self.assertEqual(active_order_status, [])
+
+    def test_check_that_order_can_be_completed(self):
+        """
+        purpose: test to check if order can be completed: active user, products on order payment type on order, 
+        payment complete = 1 
+        author: Ike
+        methods: get_ordered_products: returns a list of products
+        """
+        with sqlite3.connect('../bangazon.db') as conn:
+            cursor = conn.cursor()
+            cursor.execute("""
+                SELECT * FROM Products
+                WHERE productId = {}
+                """.format(4))
+            selected_product = cursor.fetchone()
+            cursor.close()
+
+        rfk = Customer("Robert Kennedy", '1919 Heron Pointe Dr', "Nashville", "TN", 37214, '8888978880')
```

Comment:
Some white space to break up the code into conceptual blocks, along with comments, will make this code easier to read